.github/workflows/spirv-tests.yml

```diff
@@ -23,7 +24,7 @@ jobs:
     name: Test SPIR-V
     uses: ./.github/workflows/llvm-project-tests.yml
     with:
-      build_target: check-llvm-codegen-spirv
-      projects:
-      extra_cmake_args: '-DLLVM_TARGETS_TO_BUILD="SPIRV" -DLLVM_INCLUDE_SPIRV_TOOLS_TESTS=ON'
+      build_target: check-llvm-codegen-spirv check-mlir
+      projects: mlir
+      extra_cmake_args: '-DLLVM_TARGETS_TO_BUILD="X86;SPIRV" -DLLVM_INCLUDE_SPIRV_TOOLS_TESTS=ON'
```

Comment:
Should this be 'HOST' instead?

Reply:
That does not seem to work:
```
CMake Error at CMakeLists.txt:1116 (message):
  The target `HOST' is not a core tier target.  It may be experimental, if so
  it must be passed via LLVM_EXPERIMENTAL_TARGETS_TO_BUILD.

  Core tier targets:
  AArch64;AMDGPU;ARM;AVR;BPF;Hexagon;Lanai;LoongArch;Mips;MSP430;NVPTX;PowerPC;RISCV;Sparc;SPIRV;SystemZ;VE;WebAssembly;X86;XCore


  Known experimental targets: ARC;CSKY;DirectX;M68k;Xtensa
```